### DISCUSSION TITLE
feature: Create collections/data files if they don't already exist

### DISCUSSION THREAD

**Discussion by hithismani** [decaporg/decap-cms#6900] [Ideas]
Hello!

I know at the moment Decap expects all the collections mentioned in the config file, to actually exist. 

Eg: If my config file states "content/events" as a collection folder route, it only shows up on the dashboard if the route exists.

While this approach may work for folks who like to set things up themselves, I'm wondering if we could have Decap create missing files and folders in the directory (and initialise them with default values) through maybe a key called:
`  init_missing: true`

So setting up multiple client projects becomes easier :)

**Comment by martinjagodic** [comment]
We don't want a check if files exist on every load because of performance. And we don't want to create files without user confirmation.

A viable option would be an additional page called "config/files validator" which would list missing files. You could then add a button "create missing files".

If you are willing to contribute, we would accept something like that.
